airbyte-ci/connectors/pipelines/pipelines/commands/groups/tests.py

```diff
@@ -52,14 +53,20 @@ async def run_test(poetry_package_path: str, test_directory: str) -> bool:
             pytest_container = await (
                 dagger_client.container()
                 .from_("python:3.10.12")
-                .with_exec(["apt-get", "update"])
-                .with_exec(["apt-get", "install", "-y", "bash", "git", "curl"])
-                .with_env_variable("VERSION", "24.0.2")
-                .with_exec(["sh", "-c", "curl -fsSL https://get.docker.com | sh"])
-                .with_exec(["pip", "install", "pipx"])
-                .with_exec(["pipx", "ensurepath"])
                 .with_env_variable("PIPX_BIN_DIR", "/usr/local/bin")
-                .with_exec(["pipx", "install", "poetry"])
+                .with_exec(
+                    sh_dash_c(
+                        [
+                            "apt-get update",
+                            "apt-get install -y bash git curl",
+                            "pip install pipx",
+                            "pipx ensurepath",
+                            "pipx install poetry",
+                        ]
+                    )
+                )
+                .with_env_variable("VERSION", "24.0.2")
```

Comment:
Can we use the constant that determine the docker version we use here (`DOCKER_VERSION`)?
It will make more obvious that `VERSION` means docker version :) 
https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/consts.py#L29

Reply:
Yes please